po/fr.po

```diff
@@ -52,8 +52,8 @@ msgid ""
 "Added ability to export snippets to a JSON file via `--export` or `-e` CLI "
 "options."
 msgstr ""
-"Ajouté la possibilité d'exporter des extraits vers un fichier JSON via les "
-"options en ligne de commande `--export' ou `-e'."
+"Ajout de la possibilité d'exporter des extraits vers un fichier JSON via les "
+"options en ligne de commande `--export` ou `-e`."
 
 #: data/com.github.bytepixie.snippetpixie.appdata.xml.in:22
```

Comment:
This is where the "or welcome screen" bit moved to.

Reply:
Should be fixed